.github/workflows/renovate.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         uses: renovatebot/github-action@v40.2.3
         with:
           token: '${{ steps.get_token.outputs.token }}'
+          configurationFile: "./renovate.json"
```

Comment:
`configurationFile` is for the [global config](https://docs.renovatebot.com/self-hosted-configuration/) which is a different config file than `renovate.json`. Since `postUpdateOptions` is a setting in `renovate.json`, this line should be removed

Reply:
Ok, so I found that it always reads the configuration from the default branch, so then I need to remove the variable and force-merge this PR. That sounds right?

Reply:
The global config is being read from the `env` of this action. We don't nee don't need to specify the config file. After renovate is started it reads our `renovate.json` and creates the PRs. See here:

https://github.com/kubearchive/kubearchive/actions/runs/10074716026/job/27851462762#step:5:175


We don't need `configurationFile`

Reply:
As for testing changes to renovate.json, I'm not sure how to do it. Maybe there is a renovate linter?

Reply:
it looks like this might help https://docs.renovatebot.com/config-validation/

Reply:
My claim that the configuration was not read is not true. However using `configurationFile` helped me test the behaviour of adding `goModTidy` as post-action. I will remove the parameter now that I tested it, but it will be useful in the future if we want to test behavior before merging.